sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -719,10 +721,14 @@ internal static void CheckNonZeroPvBeforeMaturityAndZeroAfter(ListAggregationRes
                 {
                     Assert.That(pv, Is.Not.EqualTo(0).Within(1e-12));
                 }
-                else
+                if (date > maturityDate)
                 {
                     Assert.That(pv, Is.EqualTo(0).Within(1e-12));
                 }
+                if (date == maturityDate)
+                { // Currently, we do not perform an assertion if the date is exactly the maturity date, as it's not clear what the PV should be.
```

Comment:
_"as it's not clear what the PV should be"_ doesnt help a user, we need to make a statement on what the user should expect

Reply:
to some degree it's a comment for us as much as it is for the user, but reworded to clarify that it's a decision we've not yet made